### PR TITLE
Add e2e test for slack join short link

### PR DESCRIPTION
**Pull request by just-at-uber**
The cypress integration test is to assert when using the slack short link it will navigate users to the slack join screen instead of the signup page.

## Screenshots
**Run test with short-link from cypress app**
Screen Shot 2020-12-11 at 1 59 45 PM

**Run test with signup page from cypress app (fail is expected here)**
Screen Shot 2020-12-11 at 2 02 04 PM

**Run from command line (will be useful for setting up github action)**
Screen Shot 2020-12-11 at 1 48 32 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .DS_Store
+cypress/fixtures
+cypress/plugins
+cypress/support
+cypress/videos
```
Comment (just-at-uber) [inline .gitignore:5]: These folders are auto-generated by cypress when running `cypress open` command. These are not modified so I have ignored these files on check-in.

File: cypress.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (just-at-uber) [inline cypress.json:1]: This file is for configuration for cypress. It is auto-generated by running `cypress open` command.

File: package.json [hunk 3]
```diff
@@ -5,7 +5,10 @@
   "license": "MIT",
   "scripts": {
     "build": "vuepress build src",
-    "start": "vuepress dev src"
+    "cypress:open": "cypress open",
```
Comment (just-at-uber) [inline package.json:8]: open command runs in cypress app (see screenshot 1 & 2)

File: package.json [hunk 4]
```diff
@@ -5,7 +5,10 @@
...
+    "cypress:run": "cypress run",
```
Comment (just-at-uber) [inline package.json:9]: run command runs in headless mode in terminal (see screenshot 3)

**Review by mkolodezny — APPROVED** [review]

**Pull request merged**